### DISCUSSION TITLE
How do I modify the name of settings?

### DISCUSSION THREAD

**Discussion by naemhui** [community/community#140051] [GitHub Education]
### Hi everyone,

There is no problem with the documents I submitted at all. I think there is a problem with the name setting of my account. Which part of the settings should I modify the name? For your information, I am Korean.
Should I modify the name of building information or should I modify the name of public profile?

**Comment by davevad93** [comment]
Hi @naemhui ,

You have to add your real full name (as it appears in your academic documentation) by clicking on `Edit Profile` in your profile page:

<img width="244" alt="0" src="https://github.com/user-attachments/assets/47ec82d7-2159-4753-b036-3317bbf46879">
<br/>
<br/>

If you have other doubts, see also the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) where you'll find a lot of useful information.
